guides/common/modules/proc_decreasing-performance-impact-of-the-pull-based-rex-transport.adoc

```diff
@@ -0,0 +1,40 @@
+[id="Decreasing_performance_impact_of_the_pull-based_REX_transport_{context}"]
+= Decreasing Performance Impact of the Pull-Based REX Transport
+
+When {ProjectServer} is configured to use REX pull-based transport for remote execution jobs using "Script" provider, {SmartProxyServer} sends a notification about new jobs to clients through MQTT.
+This notification does not include the actual workload that the client is supposed to execute.
+After a client receives a message about a new remote execution job, it queries {SmartProxyServer} for its actual payload.
+During the job, it periodically sends outputs of the job to {SmartProxyServer}, further increasing the number of requests to {SmartProxyServer}.
+
+These requests to {SmartProxyServer} together with high concurrency allowed by the MQTT protocol can cause exhaustion of available connections on {SmartProxyServer}.
+Some requests might fail, making some child tasks of remote execution jobs unresponsive. 
+This also depends on actual job payload, as some jobs are causing additional load to {ProjectServer}, making it compete for resources if clients are registered to {ProjectServer}.
+
+To avoid this, configure your {ProjectServer} and {SmartProxyServer} with the following parameters:
+
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+* MQTT Time To Live {endash} Time interval in seconds given to the host to pick up the job before considering the job undelivered
```

Comment:
Does it break the HTML @mjivraja ? Please provide a link/screenshot.

Reply:
It does not break HTML but looks unusual.

![image](https://github.com/theforeman/foreman-documentation/assets/101559972/9020d6c4-3de5-4c4d-8410-9672b8b26343)

Reply:
This is not supposed to happen. Weird. Do you know what's going on here? @Lennonka

Reply:
It's because the list was surrounded with a code block. I'm assuming this will be fixed on the next push, because I've addressed it in a [comment that is already resolved](https://github.com/theforeman/foreman-documentation/pull/2170/files#r1381907119)

Reply:
I don't think there should be periods at the ends of the items, because they aren't whole sentences.

Reply:
@Imaanpreet Please, follow up on this.